odoo/addons/base/models/ir_module.py

```diff
@@ -550,12 +550,31 @@ def _button_immediate_function(self, function):
                 "is best to write it as a standalone script, and ask the runbot/metastorm team "
                 "for help."
             )
+
+        # We must abort requested module operation if it may lead to a mess and inconsistency.
+        # Here we test that no other module operation is in progress
+        if not self.search_count([('state', 'in', ('to install', 'to upgrade', 'to remove'))], limit=1):
```

Comment:
OK, I am a blind reviewer 😭 
```suggestion
        if self.search_count([('state', 'in', ('to install', 'to upgrade', 'to remove'))], limit=1):
```

Reply:
:grimacing: